### DISCUSSION TITLE
Just some small quality tweaks to osu!

### DISCUSSION THREAD

**Discussion by RoyceThomas** [ppy/osu#16040] [Ideas] [answered]
hello, could i suggest just a couple of small quality tweaks
1. slow down or extend the animation of the opening/closing of the “translucent yellow slider threshold circle” so players can actually visualise the opening and closing animation and so it more resembles osu!stable where the circle dilates from the beat location, as opposed the rather quick animation laser currently has (well on my screen anyway)
2. ~~add a tiny background to the osu!laser version text on the bottom edge of the screen so it doesn’t mix with text when scrolling though pages like beatmaps listings~~ (https://github.com/ppy/osu/discussions/16064)

**Comment by frenzibyte** [comment, marked as answer]
Can you provide a video showcasing what you mean behind your first point? I'm not sure I completely understand it.

As for the osu!lazer version text on the bottom, I think it would make sense to hide it when an overlay is open, like it already does when opening song select / editor / multiplayer screens.

**Reply by RoyceThomas** [reply]
Sure. Its not really apparent in the video cos I have a potato computer that cant record 60fps however I'll try elaborate on it. Feel free to perform this comparison on your (most likely better) computer.
Link to potato videos: 
1.mov: https://youtu.be/KzSq2NQNfzA
2.mov: https://youtu.be/S4MJetQaWos
3.mov: https://youtu.be/JGgMJVRMjeE

The yellow slider circle that dilates from the beat location seems much more fluid and slower (which is a good thing in my opinion) in stable, as the opening and closing animation of the ring (as observed in 2.mov) is slower and the **circle remains visible** (contracting and fading) for longer **even after the slider beat has finished** (which is the attribute I want maintained in laser) (and is much more noticeable in the videos). If it is possible to observe in 1.mov, the animation of opening and closing of the same slider circle is much faster and almost instantaneously fades away as soon as the beat finishes (which I don't really like and kinda detracts the fluidity of osu!laser). Keep in mind I do know about the classic mod that removes the opening/closing animations of the yellow slider circle, however in both videos, all mods are disabled.

As for my second point, pls refer to 3.mov as I was specifically only talking about when viewing pages in the toolbar such as beatmap listing, wiki, etc, in such instances the version text does not disappear or have an observable background (sorry for the confusion there). **An edit** to my suggestion could be just **hiding the text all together when opening said pages** (beatmap listing, wiki, etc) so the text does not merge with the actual page text.

**Reply by frenzibyte** [reply]
> As for my second point, pls refer to 3.mov as I was specifically only talking about when viewing pages in the toolbar such as beatmap listing, wiki, etc, in such instances the version text does not disappear or have an observable background (sorry for the confusion there). An edit to my suggestion could be just hiding the text all together when opening said pages (beatmap listing, wiki, etc) so the text does not merge with the actual page text.

Separated to https://github.com/ppy/osu/discussions/16064.

**Reply by frenzibyte** [reply]
> The yellow slider circle that dilates from the beat location seems much more fluid and slower (which is a good thing in my opinion) in stable, as the opening and closing animation of the ring (as observed in 2.mov) is slower and the circle remains visible (contracting and fading) for longer even after the slider beat has finished (which is the attribute I want maintained in laser) (and is much more noticeable in the videos). If it is possible to observe in 1.mov, the animation of opening and closing of the same slider circle is much faster and almost instantaneously fades away as soon as the beat finishes (which I don't really like and kinda detracts the fluidity of osu!laser). Keep in mind I do know about the classic mod that removes the opening/closing animations of the yellow slider circle, however in both videos, all mods are disabled.

Just to keep things clear, this suggestion is for the osu!(lazer) default skin itself, right? If it's meant for legacy skins in osu!(lazer) then you may be looking at https://github.com/ppy/osu/discussions/13660.

**Reply by RoyceThomas** [reply]
i am referring to both default laser skin and classic. however after reading both discussions you have posted, both have been addressed.
Links for addressed posts
1: https://github.com/ppy/osu/pull/16065#issuecomment-992087569  (addressed only and no solution implemented yet (as per my knowledge)) (addressed and and fixed in [2021.1219.0](https://github.com/ppy/osu/releases/tag/2021.1218.0)))
2: https://github.com/ppy/osu/discussions/13660#discussioncomment-923124  (addressed only and no solution implemented yet (as per my knowledge))

Thanks so much [frenzibyte](https://github.com/frenzibyte)! I'm marking this discussion as answered as both my tweaks have been addressed before (peppy has addressed and frenzibyte fixed 1 while 2 remains as only addressed).

**Comment by integerrr** [comment]
For the slider follow circles, the game is doing exactly what you sees. For example, with classic mod, the slider follow circle jumps to the maximum radium once you hit the slider head, which is the same behaviour as stable, but in contrast, stable shows the follow circle  with a gradually increasing radius even though that's not what it's actually doing

**Reply by RoyceThomas** [reply]
true, i do know that the classic mod can allow that, however i was referring to the animation of this with no mods selected. feel free to read over my much more detailed response in the previous comment.
